src/utils/data_reader.py

```
@@ -0,0 +1,136 @@
+        files = self.manager.list_files('firms', f"firms_{country.lower()}_*")
+        
+        if not files:
+            raise FileNotFoundError(f"No fire data found for {country}")
+        
+        latest_file = files[-1]
+        return pd.read_csv(latest_file, parse_dates=['acq_date'])
+    
+    def read_era5_pbl(
+        self,
+        start_date: datetime,
+        end_date: datetime
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)